src/static/bitbucket/utils/RequestProxy.ts

```diff
@@ -2,25 +2,36 @@ interface BBRequest<T> {
   (opts: {
     url: string;
     type?: string;
+    data?: string;
+    contentType?: string;
     success: (resp: T) => void;
     error: (err: any) => void;
   }): void;
 }
 
+interface Data<T> {
+  mergeStrategy?: string;
```

Comment:
This is a generic request utility — it shouldn't know about specific options that will be passed to it. Consider typing it as such.